recipes/rb-pygments.rb/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pygments.rb" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 41729ecc69624bd3fc6bcc13d6ccb6ff0263334c42f66bfcf517a120addbb093
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies -n $PREFIX/bin {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
```

Comment:
I'm not sure this works as you expect. v0 recipes use `skip: True  # [win]`

Reply:
Hello @ocefpaf , Tried as suggested, but it throws error `noarch packages can't have skips with selectors. `. So reverted as before. It was taken reference from this [PR](https://github.com/conda-forge/staged-recipes/pull/29315/files#diff-e7f7cd3a52034068d71927075bc9523f5877a11a843c2b28edec3f1a09bb71d4). Shall we keep in this format. Kindly have a look again

Reply:
If you are going for a noarch route, you can't skip Windows here.

It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.

Reply:
@ocefpaf , Thanks for the clarification. `win` build is enabled now. Kindly have a look again